nautobot_ssot/integrations/infoblox/utils/client.py

```diff
@@ -791,7 +791,8 @@ def get_all_subnets(self, prefix: str = None):
             "_return_fields": "network,network_view,comment,extattrs,rir_organization,rir,vlans",
             "_max_results": 10000,
         }
-
+        if PLUGIN_CFG.get("NAUTOBOT_INFOBLOX_NETWORK_VIEW"):
```

Comment:
Should the inclusion of the `network_view` be here as well? https://github.com/nautobot/nautobot-plugin-ssot/blob/2a0f24d0612cbe934629d666a4ef9113cfa9e71b/nautobot_ssot/integrations/infoblox/utils/client.py#L1278

Reply:
Do you mean include the check for the Network View in the `if prefix` statement instead of just above it? There is check for the Newtwork View just before this.

Reply:
Sorry, I should've been more clear. You added the check for the network view in `get_all_subnets` do we also need that check in `get_network_containers` so that when `get_network_containers` gets called we are only retrieving network containers within the specified network view?

Reply:
There is a check for the Network View in these functions: `get_all_subnets`, `get_network_containers`, `get_child_network_containers` and `get_child_subnets_from_container`. I think the check you are referring to is already included on the new line 1288.

Reply:
Ah yep! Okay, I think I was just looking too fast at the code. Thanks @Dav-C!